app/assets/stylesheets/sass/abstracts/_mixins.css.scss

```diff
@@ -0,0 +1,11 @@
+@mixin center-flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
sort this block

Reply:
@JuanequeX. Just to be sure, when you say sort, do you mean order the properties alphabetically? It's the first time I hear this

Reply:
@Frcomparan, indeed, a good practice is to keep the properties of the style sheet classes in order.

Reply:
Okay, I have sorted the properties of the CSS files